chapters/annotations.tex

```diff
@@ -2324,6 +2324,18 @@ \subsection{Licensing}\label{licensing}
 \end{example}
 
 
+\section{License Texts}\label{license-texts}
+
+For a top-level class the {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives a license file for the class.
+The annotation may give a scalar value or an array of values.
+The license files are human-readable files containing the license conditions.
+If the annotation specifies an array of license files the conditions in all of them must be satisfied (i.e., it is not intended for dual-licensing).
+External functions may also contain this annotation (with similar content), see \cref{annotations-for-external-libraries-and-include-files}.
+When generating a distributable version for a translated model the license files for used functions and top-level classes should be included.
```

Comment:
New paragraph to make exception and the final sentence more clearly referring to the _should_:
```suggestion

When generating a distributable version for a translated model the license files for used functions and top-level classes should be included.
```